docs/deep-dives/github.md

```diff
@@ -43,8 +43,33 @@ steps:
 ```
 
 With Docker Compose,
-declare the environment variables and volume mounts in your [Compose file]:
+declare volume mounts in your [Compose file].
 
+You won't need to declare the environment variables in the compose file since `propagate-environment: true` will ensure the variables defined are propagated to the docker environment.
+
+For example, with the [Docker Buildkite plugin], we will need to add the `propagate-environment: true`:
```

Comment:
```suggestion
For example, with the [Docker Compose Buildkite plugin]:
```

Reply:
what should this link to? there is no `[Docker Compose Buildkite plugin]` doco linked down here: https://github.com/seek-oss/skuba/blob/master/docs/deep-dives/github.md?plain=1#L123

Reply:
I think here: https://github.com/buildkite-plugins/docker-compose-buildkite-plugin

Reply:
updated